PubSubCentralized/PubSub.Publisher/Program.cs

```diff
@@ -10,52 +10,47 @@ class Program
     {
         static void Main()
         {
-            using (var activator = new BuiltinHandlerActivator())
-            {
-                Configure.With(activator)
-                    .Logging(l => l.ColoredConsole(minLevel: LogLevel.Warn))
-                    .Transport(t => t.UseMsmq("publisher"))
-                    .Subscriptions(s => s.StoreInSqlServer("server=.; database=RebusPubSubCentralized; trusted_connection=true", "Subscriptions", isCentralized: true))
-                    .Start();
+            var bus = Configure.OneWayClient()
```

Comment:
While it's definitely not as important as with Rebus instances capable of receiving stuff, one should still dispose the bus when the app shuts down. In this case, it will be pretty with a simple using statement
```csharp
using var bus = Configure.OneWayClient()
    .(...)
```